### DISCUSSION TITLE
Function documentation other than beginning and advanced tutorials?

### DISCUSSION THREAD

**Discussion by BrighamEaquinto** [fastapi/fastapi#9597] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI, Path
```

### Description

Is there documentation on the FastAPI classes/modules/functions/function parameters? I enjoy using the tutorials, but I'd like to see the specifics of each function and function parameter, like the `Path()` function from the fastapi module, for example. 

In a Pandas example, if I needed more information about making a dataframe, I'd go here for the additional information -> https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html

Is there an equivalent of this? The closest I can find is going to the source code, but there is no description for the specifics of everything-> https://github.com/tiangolo/fastapi/blob/master/fastapi/applications.py#L440

### Operating System

Linux

### FastAPI Version

0.95.2

### Python Version

3.8.10

**Comment by nymous** [comment, marked as answer]
Hello!
You can follow this (issue converted to) discussion: https://github.com/tiangolo/fastapi/discussions/9153
